resources/public/api/conf/6.0/foreign_property_retrieve.yaml

```diff
@@ -12,33 +11,39 @@ get:
-    | FOREIGN_NON_FHL_PROPERTY_ALLOWANCE            | Simulates success response with non FHL adjusted summary and propertyAllowance.                                                          |
-    | DYNAMIC_FOREIGN_NON_FHL_PROPERTY_ALLOWANCE    | Simulates success response with non FHL adjusted summary and propertyAllowance. <br><br> See below for details on DYNAMIC scenarios.     |
-    | FOREIGN_PROPERTY_STATUS_INVALID               | Simulates success response with summary status invalid.                                                                                  |
-    | DYNAMIC_FOREIGN_PROPERTY_STATUS_INVALID       | Simulates success response with summary status invalid. <br><br> See below for details on DYNAMIC scenarios.                             |
-    | FOREIGN_PROPERTY_STATUS_SUPERSEDED            | Simulates success response with summary status superseded.                                                                               |
-    | DYNAMIC_FOREIGN_PROPERTY_STATUS_SUPERSEDED    | Simulates success response with summary status superseded. <br><br> See below for details on DYNAMIC scenarios.                          |
-    | NOT_FOREIGN_PROPERTY                          | Simulates the scenario where the calculationId is not a foreign property.                                                                |
-    | TAX_YEAR_NOT_SUPPORTED                        | Simulates the scenario where the tax year is not supported.                                                                              |
-    | NOT_FOUND                                     | Simulates the scenario where no matching resource is found.                                                                              |
-    | REQUEST_CANNOT_BE_FULFILLED                   | Simulates the scenario where the request cannot be fulfilled for various reasons other than the ones triggering other errors.            |
+    |  Tax Year                    | Header Value (Gov-Test-Scenario)              | Scenario                                                                                                                                 |
+    | -----------------------------| --------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+    | All                          | N/A - DEFAULT                                 | Simulates the scenario where no data is found.                                                                                           |
+    | ALL                          | FOREIGN_PROPERTY_STATUS_INVALID               | Simulates success response with summary status invalid.                                                                                  |
```

Comment:
Minor: these should probably all be "All" rather than "ALL"

Reply:
Done